jwst/associations/lib/rules_level3.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, *args, **kwargs):
             DMSAttrConstraint(
                 name='wfsvisit',
                 sources=['visitype'],
-                value='.+wfsc.+',
+                value='prime_wfsc_sensing_control',
```

Comment:
I thought there were 2 or 3 different values of VISITYPE that pertain to WFSC (have the substring "WFSC" in them).

Reply:
Further investigation has revealed that it is only this particular visit that will be producing the dual imaging. Again, will need a real run, WFR#2 maybe, to confirm.